### ISSUE TITLE
requireNativeComponent is not a function with react-native-web

### ISSUE THREAD

**Issue by penx** [callstack/react-native-slider#185]
## Environment

react-native-web, storybook

## Description
"requireNativeComponent is not a function" when using 3.0.0-rc.2 and react-native-web

## Reproducible Demo

https://github.com/penx/react-mixer/tree/6126b45efefb9bd4b4e600d9a4e6191616d7a8c3

Steps to recreate:

- `npm init -y`
- `npx -p @storybook/cli sb init --type react`
- add react-native-web to dependencies and in storybook webpack alias
- add `import Slider from '@react-native-community/slider';` to a story

```
TypeError: (0 , _reactNative.requireNativeComponent) is not a function
    at Object../node_modules/@react-native-community/slider/dist/RNCSliderNativeComponent.js (http://localhost:6006/vendors~main.164a2871296ea09f7f00.bundle.js:115:255)
```

**Comment by arthurpankiewicz** [comment]
I'm also running into this problem with a few tests, particularly 
```
react-native-safe-area-context
react-native-calendars
react-native-gesture-handler
```
I'm thinking this might be more of a `react-native-web` problem though

**Comment by haveamission** [comment]
Has any solution been found for this? I cannot use React Native Slider in React Native Web.

**Comment by carltian** [comment]
+1

**Comment by younesshakky** [comment]
any news on this one? @penx did you resolve it somehow?

**Comment by ajaybandi14** [comment]
Any solution for this yet?

**Comment by BartoszKlonowski** [comment]
Yes, this one can be closed by #462
Thank you, @jspizziri!
